Switch to SkyBiometry Face Detection and Recognition API

Liuftvafas opened the pull request:
SkyBiometry Face Detection And Recognition API is intended to be used as a drop-in replacement for developers using face.com API. I would like to suggest to switch to this service because face.com API is no longer available.

Liuftvafas committed:
Replaced endpoint to api.skybiometry.com/fc.

Because SkyBiometry is intended to be used as a drop-in replacement for
developers using face.com API most of functionality should work.

Liuftvafas committed:
Changed links to documentation and account.

Liuftvafas committed:
Updated links.

Liuftvafas committed:
Switch EOLs to UNIX/OSX.

Kami commented:
Sorry for the delay.

Switching to a different (compatible) API seems reasonable although it should probably be a separate library because this one has "face" in the name which would make it confusing.

Liuftvafas commented:
Pros:
1. It's the same API just the different endpoint.
2. Face can refer to human face. That's what this library is about.
Cons:
???

Liuftvafas committed:
Updated changes and moved examples to readme.

Liuftvafas committed:
Added real response from SkyBiometry service.

Liuftvafas committed:
Bold on uids list.

Liuftvafas committed:
Added real responses from SkyBiometry service. Fixed tags_get() arguments processing.

Liuftvafas committed:
Fixed namespace name in responses.

Liuftvafas committed:
Minor changes.

Liuftvafas committed:
Changes tabs to spaces.

Liuftvafas committed:
Minor changes.

Liuftvafas committed:
Minor changes.

JimJty committed:
Update face_client.py

Some minor fixes:
- removed print statement
- added account/namespaces method
- 'detector' parameter was missing in faces/recognize call

Liuftvafas committed:
Merge pull request #1 from JimJty/patch-1

Update face_client.py

Liuftvafas committed:
Fixed whitespace and links to documentation.

Liuftvafas committed:
Added reading of response in case of HTTPError.

Commit:
Added Python 3 support

Liuftvafas committed:
Merge pull request #3 from jsutas999/master

Added Python 3 support

Matas-NT committed:
UPD: Multipart request Python 3.X compatibility.
